x-pack/plugins/canvas/common/lib/httpurl.js

```diff
@@ -7,6 +7,6 @@
 // A cheap regex to distinguish an HTTP URL string from a data URL string
 const httpurlRegex = /^https?:\/\/\S+(?:[0-9]+)?\/\S{1,}/;
 
-export function isValid(str) {
+export function isValidHttpUrl(str) {
```

Comment:
Why did you rename this export? Was it required for some reason?

Reply:
To avoid the collision names as we are exporting everything 😃